docs/src/components/home/sections/HeroSection.tsx

```diff
@@ -76,37 +77,29 @@ export const HeroSection = () => {
             components that can connect directly to the cloud.
           </Text>
           {platform === 'react' ? (
-            <Flex direction={['column', 'column', 'row']}>
-              <HomeFeatureCard
+            <Grid gap="medium" templateColumns={['1fr', '1fr', '1fr 1fr 1fr']}>
```

Comment:
I think we could achieve the same but there is not much reason to use CardLinkGroup unless you want the default gap + templateColumns and want to add the title. Since this doesn't use either I just left it as a Grid element.